executor/executor.go

```diff
@@ -121,6 +122,7 @@ func newBaseExecutor(ctx sessionctx.Context, schema *expression.Schema, id strin
 		ctx:          ctx,
 		id:           id,
 		schema:       schema,
+		initCap:      ctx.GetSessionVars().MaxChunkSize,
```

Comment:
In this PR, we can not do that, because JOIN relate modification will in next PR.

then we will make initCap be configurable and check constrait in set config stage.